lib/rucio/rse/protocols/protocol.py

```diff
@@ -41,16 +41,31 @@ class RSEDeterministicScopeTranslation(PolicyPackageAlgorithms):
     """
         Translates a pfn dictionary into a scope and name
     """
+
+    _algorithm_type = "pfn2scope_name"
+
     def __init__(self, vo: str = 'def'):
         super().__init__()
         self.register("def", RSEDeterministicScopeTranslation._default)
         self.register("atlas", RSEDeterministicScopeTranslation._atlas)
         policy_module = vo
-        # Uses the same policy as the DeterministicTranslation
-        if super()._supports(self.__class__.__name__, policy_module):
-            self.parser = self._get_one_algorithm(self.__class__.__name__, policy_module)
-        else:
-            self.parser = self._get_one_algorithm(self.__class__.__name__, "def")
+        try:
+            # Use the function defined in the policy package if it's configured so
+            algo_type = self._algorithm_type
+            algorithm_name = config.config_get('policy', self._algorithm_type)
+        except (NoOptionError, NoSectionError, RuntimeError):
```

Comment:
What causes this runtime error, is it just a missing config? It's a little generic, it could be useful to put a logging statement here just in case.

Reply:
If the configuration is missing,  we should get `NoOptionError` or `NoSectionError` exceptions. I actually don't know what would throw `RuntimeError`. I've included it, since it was also included in many places in rucio code where a config is tried to be read, e.g. [1]. Not sure if we should keep it or not.

I can put a logging line. Should I make it `DEBUG` level? For communities that don't rely on policy package for this functionality, it might clutter their logging o/w

[1] https://github.com/rucio/rucio/blob/a62dfe18216cd0bcea238b5ed7b1197345a7f2fd/lib/rucio/daemons/reaper/reaper.py#L274

Reply:
I would do it at debug level for sure, if we don't know what the Runtime is catching I would mark that as better safe than sorry. It is up to your digression though.

Reply:
I added the logging and I kept the RuntimeError exception. Does it look good now?